### PR TITLE
Add Login page

### PR DESCRIPTION
**Pull request by Bicik89**
As discussed with Piotr, used new atom FieldInput

### CODE REVIEW

File: src/components/organisms/FormLogin.vue [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    const v$ = useVuelidate();
+    return { v$ };
+  },
+  methods: {
+    async submitForm() {
+      this.v$.$validate();
+      if (this.v$.$error) {
+        return null;
+      }
+      try {
```
Comment (242370) [inline src/components/organisms/FormLogin.vue]: Change to Promise style exceptions handling

File: src/components/organisms/FormLogin.vue [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        const response = await axios.post(GATEWAY_ADDRESS + '/login', { email: this.email, password: this.password });
+        console.log('User logged in: ', response.data);
+        // TODO where to redirect
```
Comment (242370) [inline src/components/organisms/FormLogin.vue]: Redirect to the main page

File: src/components/organisms/FormLogin.vue [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+  components: { FieldInput, ButtonPrimary },
+  data() {
+    return {
+      email: "",
+      password: "",
+      error: "",
+      text: "Zaloguj się"
+    }
+  },
+  // here also might want to move it
```
Comment (242370) [inline src/components/organisms/FormLogin.vue]: Remove such comments

**Review by 242370 — COMMENTED** [review]
Also resolve merge conflicts

**Review by Piotr-Skrobski — APPROVED** [review]

**Pull request merged**